document/html/index.bs

```diff
@@ -0,0 +1,389 @@
+<pre class='metadata'>
+Title: WebAssembly Web API
+H1: WebAssembly
+Shortname: wasmweb
+Group: WebAssembly CG
+Status: w3c/CG-DRAFT
+Level: 1
+ED: https://webassembly.github.io/spec/html/index.html
+Editor: WebAssembly Community Group
+Repository: WebAssembly/spec
+Abstract: This document describes the integration of WebAssembly with the broader Web Platform.
+Markup Shorthands: css no, markdown yes
+Ignored Terms: h1, h2, h3, h4, h5, h6, xmp
+</pre>
+
+<pre class='biblio'>
+{
+  "ECMA-262": {
+    "href": "https://tc39.github.io/ecma262",
+    "title": "ECMAScript® 2018 Language Specification",
+    "publisher": "ECMA TC39",
+    "status": "Current Editor's Draft"
+  },
+  "WEBASSEMBLY": {
+    "href": "https://webassembly.github.io/spec/",
+    "title": "WebAssembly Specification",
+    "publisher": "W3C WebAssembly Community Group",
+    "status": "Draft"
+  },
+  "WASMJS": {
+    "href": "https://littledan.github.io/spec/js-api/",
+    "title": "WebAssembly JS Integration Specification",
+    "publisher": "W3C WebAssembly Community Group",
+    "status": "Draft"
+  }
+}
+</pre>
+
+<pre class="anchors">
+urlPrefix: https://tc39.github.io/ecma262/; spec: ECMA-262
```

Comment:
You may want to consider pulling the shared anchors out into a separate file instead of duplicating them in both specs. See https://tabatkins.github.io/bikeshed/#custom-dfns last paragraph